index.html

```diff
@@ -18,21 +18,38 @@
 
 <body>
     <header class="header">
-        <div class="logo">
-            <a href="index.html">My Logo</a>
-        </div>
-        <nav>
-            <div class="humberger">
-                <i class="fa fa-bars"></i>
+        <div class="headergrid">
+            <div class="logo">
+                <a href="index.html">My Logo</a>
             </div>
-        </nav>
+            <nav>
+                <ul class="desktopmenu">
+                    <li>
+                        <a href="#">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="">ABout</a>
+                    </li>
+                    <li>
+                        <a href="#">contact</a>
+                    </li>
+                </ul>
+                <div class="humberger bars">
+                    <i class="fa fa-bars"></i>
+                </div>
+            </nav>
+        </div>    
     </header>
     <section class="intro">
```

Comment:
- [ ] kindly consider making sure that the background image for this section covers the whole section as given in the Figma design  to make it an exact duplicate of the given design check the images below  for more clarification 
- your design 👇 
<img width="1381" alt="Screen Shot 2023-05-12 at 15 11 19" src="https://github.com/MPDADDY/portfolio/assets/60146030/0883208b-a3f1-45d4-84ae-d8523a43e3f3">


- Figma design 👇 
<img width="813" alt="Screen Shot 2023-05-12 at 15 13 40" src="https://github.com/MPDADDY/portfolio/assets/60146030/f33104e1-2aa5-41c6-a4b4-47b3ae6df58b">